airflow/providers/amazon/aws/operators/glue.py

```diff
@@ -75,10 +75,10 @@ def __init__(
         *,
         job_name: str = 'aws_glue_default_job',
         job_desc: str = 'AWS Glue Job with Airflow',
-        script_location: Optional[str] = None,
+        script_location: str,
```

Comment:
Type in docstring doesn't match.

Reply:
I have a change coming that fixes this - since we upgraded Sphinx-autoapi recently we can now use type-hints in docs, meaning we can remove all the `:type` lines!

Reply:
OH YEAH!

Reply:
https://github.com/apache/airflow/pull/20951